experiments/crossing_validation.py

```diff
@@ -0,0 +1,130 @@
+from algorithm.data_struct import *
+from algorithm.tree import build_tree, c45
+import json
+import random
+import matplotlib.pyplot as plt
+
+def test_crossing_validation(acteptors_examples: [Example], donors_examples: [Example]):
+    k = 10
+    iteration = 100
+    acteptors_results = []
+    donors_results = []
+    acteptors_results_id3 = []
+    donors_results_id3 = []
+    
+    a_result = 0
+    d_result = 0
+    a_result_id3 = 0
+    d_result_id3 = 0
+    for i in range(0, iteration):
+        a_result += crossing_validation(acteptors_examples, k)
+        d_result += crossing_validation(donors_examples, k)
+        a_result_id3 += crossing_validation(acteptors_examples, k, "id3")
+        d_result_id3 += crossing_validation(donors_examples, k, "id3")
+        
+    acteptors_results.append(((str)(k), a_result/iteration))
+    donors_results.append(((str)(k), d_result/iteration))
```

Comment:
trochę dużo tych zmiennych.... i po co robić tablicę z jednym elementem ? nie łatwiej żeby save results przyjmował tupla (k , result)

Reply:
masz rację. To zostało jeszcze po tym jak było wiele różnych k